src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -44,7 +45,8 @@ const RenderListFeed = (props) => {
     userId,
     onPressDownVote,
     selfUserId,
-    onPressUpvote
+    onPressUpvote,
+    offset
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `offset` is introduced but it's not clear from the provided context how it's being used. Ensure that it's properly validated and handled in the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->